x-pack/legacy/plugins/monitoring/common/formatting.js

```diff
@@ -6,21 +6,32 @@
 
 import moment from 'moment-timezone';
 
+/**
+ * Kind of a bad practice to depend on anything in the "/public" directory,
+ * but this way we avoid passing/prop-drilling injector all over our components
+ */
+import chrome from '../public/np_imports/ui/chrome';
```

Comment:
This seems like the wrong approach. I don't think we should be importing `public/` files into the `common/` directory. Let's try and just pass it through the components - it's only going to affect a few of them.

Reply:
💯  I agree as I have mentioned that in the comment above. I'll pass the tz as a param then